addon-test-support/ilios-common/page-objects/components/instructor-selection-manager.js

```diff
@@ -16,7 +16,11 @@ export default {
   instructors: collection('[data-test-instructors] li', {
     remove: clickable()
   }),
-  instructorGroups: collection('[data-test-instructor-groups] li', {
-    remove: clickable()
+  instructorGroups: collection('[data-test-instructor-group]', {
+    title: text('[data-test-instructor-group-title]'),
+    members: collection('[data-test-instructor-group-members] li', {
+      text: text(),
```

Comment:
`text` is actually a default attribute or every page object once it is selected. If you just want the text of the entire element this can be omitted.

Reply:
good to know. thanks! fixed.